README.md

```diff
@@ -11,14 +12,15 @@ This environment includes the latest versions of the following software:
 
 ## Prerequisites
 
- - Docker (https://www.docker.com/products/docker-desktop)
- - Tested on macOS 10.15 (Catalina) and Windows 10 Version 10.0.18362
- - Windows 10 requires Docker File Sharing with the C drive to be enabled in the Docker settings, restart may be required after enabling sharing
+ - Docker - https://www.docker.com/products/docker-desktop
+ - Tested and confirmed working on:
+	 - macOS 10.15 (Catalina)
+	 - Windows 10 Version 21H1 build 19043.1165
 
 ## Running the Container
 
 1. Using composer: `composer create-project jlucki/docker-php-dev-env <project name>` or download the files from this repo and copy them to your project directory
-3. Run `docker-compose up` from the project directory
+3. Run `docker compose up` from the project directory
```

Comment:
I'd add a note here to use `docker-compose up` on versions older than 3.6, `docker compose` (na dash), is only availble on 3.6 and up https://docs.docker.com/compose/cli-command/